src/app/Shared/Services/Api.service.tsx

```diff
@@ -1295,6 +1302,15 @@ export class ApiService {
         }`,
       { id: target.id! },
     ).pipe(
+      tap((resp) => {
+        if (resp.data == undefined) {
```

Comment:
Current example of a failing MBean metrics response:

```json
{
  "errors": [
    {
      "message": "Client Error (427)",
      "locations": [
        {
          "line": 1,
          "column": 105
        }
      ],
      "path": [
        "targetNodes",
        0,
        "target",
        "mbeanMetrics"
      ],
      "extensions": {
        "classification": "DataFetchingException",
        "exception": "io.vertx.ext.web.handler.HttpException"
      }
    }
  ],
  "data": {
    "targetNodes": [
      {
        "target": {
          "mbeanMetrics": null
        }
      }
    ]
  }
}
```

I think this should check for a condition like `resp.errors && Array.isArray(resp.errors) && resp.errors.length > 0`, rather than `resp.data == undefined`.